### PR TITLE
Updated docs on marathon's healtcheck behavior

### PR DESCRIPTION
**Pull request by solarkennedy**

### CODE REVIEW

File: docs/source/yelpsoa_configs.rst [hunk 1]
```diff
@@ -281,12 +281,23 @@ instance MAY have:
...
+`mesos documentation <https://mesos.readthedocs.io/en/latest/health-checks/>`_
+for more low-level details:
 
-  * ``healthcheck_mode``: One of ``cmd``, ``tcp``, ``http``, or ``https``. If your
-    service uses Smartstack, then this must match the value of the ``mode`` key
-    defined for this instance in ``smartstack.yaml``. If set to ``cmd`` then
-    PaaSTA will execute ``healthcheck_cmd`` and examine the return code.
+  * ``healthcheck_mode``: One of ``cmd``, ``tcp``, ``http``, or ``https``.
+    If set to ``http`` or ``https``, a ``curl`` command will be executed
+    inside the container. ``curl`` must be installed in the container for
```
Comment (Rob-Johnson) [inline docs/source/yelpsoa_configs.rst]: I don't think this is true: you need curl on the host, and it mesos executes the cmd inside the network namespace of the container.

File: docs/source/yelpsoa_configs.rst [hunk 2]
```diff
@@ -281,12 +281,23 @@ instance MAY have:
...
+    this to work.
+
+    If set to ``cmd`` then PaaSTA will execute ``healthcheck_cmd`` and
+    examine the return code. It must return 0 to be considered healthy.
+
+    If the service is registered in SmartStack, he healthcheck_mode will
```
Comment (Rob-Johnson) [inline docs/source/yelpsoa_configs.rst]: s/he/the

File: docs/source/yelpsoa_configs.rst (hunk 1 above)
Comment (solarkennedy) [reply docs/source/yelpsoa_configs.rst]: Huh. I thought for sure this was the case, but I can't find any evidence of it anymore.

File: docs/source/yelpsoa_configs.rst (hunk 2 above)
Comment (solarkennedy) [reply docs/source/yelpsoa_configs.rst]: Damn by broken T key. I'm getting a replacement laptop soon.

**Pull request merged**